src/components/SpaceBg/SpaceBg.tsx

```diff
@@ -0,0 +1,265 @@
+import React, { FC } from 'react'
+import styled, { keyframes } from 'styled-components'
+
+import HeroImage from '../../assets/images/hero-graphic-desktop.png'
+import HeroImageLeft from '../../assets/images/hero-graphic-left.png'
+import { breakpoints, gradients } from '../../utils/theme'
+
+export const SpaceBg: FC = ({ children }) => (
+  <StyledHero id="liquidity-hero" className="hero-active">
+    <div className="inner-hero">
+      <AppBodyContainer>{children}</AppBodyContainer>
+      <div className="hero-background">
+        <div className="hero-image hero-image-right"></div>
+        <div className="hero-image hero-image-left"></div>
+      </div>
+    </div>
+  </StyledHero>
+)
+
+const AppBodyContainer = styled.section`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  z-index: 3;
+  min-height: calc(100vh - 340px);
+`
+
+const zoomOut = keyframes`
+    0% {transform: scale(1.2); opacity: 0;}
+    100% {transform: scale(1); opacity: 0.7;}
+`
+
+const StyledHero = styled.div`
+    position: relative;
+    width: 100% !important;
```

Comment:
is `important` needed here?

Reply:
Fixed. Btw, we are still using "old" styles in the background space that's why there still using classes inside of styled components. Maybe we should create an issue to fix this.

Reply:
Issue created https://github.com/levelkdev/swapr-dapp/issues/1096